opensplice_cmake_module/env_hook/opensplice.sh.in

```diff
@@ -1,9 +1,15 @@
 export OSPL_HOME=@OpenSplice_HOME@
+DEFAULT_OSPL_URI=file://$OSPL_HOME/etc/opensplice/config/ospl.xml
```

Comment:
Please use an underscore as a prefix to the variable so that it does not override a user defined variable.

Also `unset` the temporary variable afterwards.